src/pages/Home/Home.jsx

```diff
@@ -1,5 +1,62 @@
 import React from "react";
+import { Link } from "react-router-dom";
+import { Card, Footer, Header } from "../../components";
+import { banner2 } from "../../data/image/productImages";
+
+import { categories } from "../../backend/db/categories";
+import { products } from "../../backend/db/products";
+import "./Home.css";
+import { useDocumentTitle } from "../../customHooks/useDocumentTitle";
 
 export const Home = () => {
-	return <div>Home Page</div>;
+	useDocumentTitle("Home | MS");
+
+	return (
+		<div className="body--homepage">
+			<Header />
+			<main className="main--homepage">
+				<section className="homepage__section categories">
+					<h2 className="h2 section__heading">Product Categories</h2>
+
+					<ul className="categories__items">
+						{categories.map((prodCat) => (
+							<li
+								key={prodCat._id}
+								style={{ backgroundColor: "white" }}
+								className="categories__list"
+							>
+								<Link to="/productlisting" className="categories__links">
+									<img
+										style={{ backgroundColor: "white" }}
+										src={prodCat.src}
+										alt="bat"
+										className="image--responsive"
+									/>
+									<span className="catergories__name">
+										Cricket {prodCat.categoryName}
+									</span>
+								</Link>
```

Comment:
You can create a pure component for prodCat and use it here.

Reply:
Sure will do it.

Reply:
Thanks for Review